__tests__/Unit/Components/Tasks/TasksContent.test.tsx

```diff
@@ -165,4 +194,70 @@ describe('tasks content', () => {
             },
         });
     });
+
+    test('should show status-tabs-container after 450px of screen width', async () => {
+        setWindowInnerWidth(breakpointToShowTabs);
+        const mockPushFunction = jest.fn();
+        renderWithRouter(
+            <Provider store={store()}>
+                <TasksContent dev={true} />
+            </Provider>,
+            { push: mockPushFunction }
+        );
+
+        await screen.findByTestId('tabs');
+        const tabsContainer = screen.getByTestId('status-tabs-container');
+        const tabStyles = getComputedStyle(tabsContainer);
+        expect(tabStyles.display).toBe('block');
+    });
+
+    test('should show status-select-container 450px below status-tabs-container', async () => {
+        setWindowInnerWidth(breakpointToShowSelect);
+        const mockPushFunction = jest.fn();
+        renderWithRouter(
+            <Provider store={store()}>
+                <TasksContent dev={true} />
+            </Provider>,
+            { push: mockPushFunction }
+        );
+        await screen.findByTestId('status-select-container');
+        const selectContainer = screen.getByTestId('status-select-container');
+        const selectStyles = getComputedStyle(selectContainer);
+
+        expect(selectStyles.display).toBe('block');
+    });
+
+    test('Selecting a value from dropdown pushes into query params', async () => {
+        setWindowInnerWidth(breakpointToShowSelect);
+        const mockPushFunction = jest.fn();
+        renderWithRouter(
+            <Provider store={store()}>
+                <TasksContent dev={true} />
+            </Provider>,
+            { push: mockPushFunction }
+        );
+
+        await screen.findByTestId('status-select-container');
+        const selectContainer = screen?.getByTestId(
+            'selected-option-container'
+        );
+
+        fireEvent.click(selectContainer);
+        fireEvent.keyDown(selectContainer, {
+            key: 'ArrowDown',
+            code: 'ArrowDown',
+        });
+
+        fireEvent.keyDown(selectContainer, {
+            key: 'Enter',
+            code: 'Enter',
+        });
+
+        expect(mockPushFunction).toBeCalledTimes(1);
+        expect(mockPushFunction).toBeCalledWith({
+            query: {
+                section: 'assigned',
```

Comment:
If the order of tabs changes, this wouldn't still be 'assigned' right? Can we please use the `Tabs` enum to select the 2nd option?

Reply:
Yes you're right made it dynamic now by using the tabs enum